homeassistant/components/homekit/__init__.py

```diff
@@ -459,3 +468,21 @@ def stop(self, *args):
 
         _LOGGER.debug("Driver stop")
         self.hass.add_job(self.driver.stop)
+
+
+class HomeKitParingQRView(HomeAssistantView):
```

Comment:
Don't use a custom HTTP view. Instead:

https://github.com/home-assistant/core/blob/0d598dd6d60d6f40ff640e0b3387d5f451b5300c/homeassistant/auth/mfa_modules/totp.py#L227-L234

https://github.com/home-assistant/core/blob/0d598dd6d60d6f40ff640e0b3387d5f451b5300c/homeassistant/components/auth/strings.json#L8

Reply:
@balloob https://github.com/home-assistant/frontend/issues/5576. I can't use an inline image because its stripped by the XSS checks

Reply:
<img width="454" alt="Screen Shot 2020-04-21 at 1 10 20 PM" src="https://user-images.githubusercontent.com/663432/79898858-8d490d80-83d1-11ea-99f1-6752850bdd2b.png">
SVG looks like this.

Reply:
@balloob ok to merge as-is?